documents-d-archeologie-francaise.csl

```diff
@@ -105,7 +106,12 @@
   </macro>
   <macro name="responsability">
     <choose>
-      <if variable="author editor translator collection-editor" match="any">
+      <if variable="version">
+        <group suffix=" (éd.)">
```

Comment:
I don't quite understand -- why are all the labels in the style set as suffix? That seems rather awkward, but I may be missing a reason this is done?